### ISSUE TITLE
Fatal error: Cannot access protected property SelectorField::$types

### ISSUE THREAD

**Issue by trabuccocampos** [storypioneers/kirby-selector#19]
This issue appears only on the live site and not in my local version (see screenshots below). I'm running Kirby 2.1 and as a solution have updated both Kirby and Selector to the latest versions. Still not working.

Below is the blueprint. Thanks for the help!

```
<?php if(!defined('KIRBY')) exit ?>

title: Team Member
pages: false
files: false
fields:
  title:
    label: Name
    type:  text
  text:
    label: Bio
    type:  textarea
  postimage:
    label: Main Post Image
    type:  selector
    mode:  single
    types:
      - image
  role:
    label: Role
    type:  text
    icon: video-camera
```
## Error on live site

![live_site](https://cloud.githubusercontent.com/assets/4658734/8804981/c819d3f6-2f9c-11e5-8731-7d23c850d4b4.png)
## Working well on local site

![local_site](https://cloud.githubusercontent.com/assets/4658734/8804984/c96e1988-2f9c-11e5-9d95-2f7232fb90fa.png)

**Comment by JonasDoebertin** [comment]
Seems to be an issue that affects only some PHP version (below 5.4, I'd guess). Could you tell me which versions you're using on both your local and your live site?

**Comment by trabuccocampos** [comment]
You nailed it: local PHP version 5.5, site version was 5.3. I changed it to 5.5 and the selector now works!
Thanks so much.

**Comment by zakgreene** [comment]
Hi, I had the same issue, and had to set my dev server to use PHP 5.5. However, for the production site, the client uses a host that only supports 5.4. Has anyone been able to confirm that 5.4 works?

**Comment by JonasDoebertin** [comment]
The error referenced in the initial comment was fixed with the [1.4.2 release](https://github.com/storypioneers/kirby-selector/releases/tag/1.4.2). It should be fully compatible with PHP 5.3. Plase make sure your version of the Selector is up to date.
